parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -247,7 +247,7 @@ ensureRunc() {
 
     TARGET_VERSION=${1:-""}
     if [[ -z ${TARGET_VERSION} ]]; then
-        TARGET_VERSION="1.0.3"
+        TARGET_VERSION="1.1.5+azure-ubuntu${UBUNTU_RELEASE}u1"
```

Comment:
does the check on L268 work with this version? 

the check below uses the runc version itself, not the apt package version.

Reply:
also L260?

Reply:
L260 is a potentially unnecessary check, since previously the runc version and apt package versions were the same. So having that to prevent older versions being used is right.

Keeping that check for now, since it's a large jump in versions.

Reply:
Will update check in L268 though